### PR TITLE
Amls 4991 - Paragraph and hint text adjustments

### PR DESCRIPTION
**Pull request by slavvco**
Made some hints paragraphs and opposite as described in ticket. There is list with affected pages.

## Related / Dependant PRs?

- none

## How Has This Been Tested?

- manually 
- run unit tests

## Checklist

- [ ] Breaking change (fix or feature that would cause existing functionality to change).
- [x] Build passing.
- [ ] Unit tests have full coverage.
- [ ] Unit test approach and implementation has been reviewed with QA (testers).
- [ ] Requires acceptance test run.
- [x] Appropriate labels added.
- [x] RELEASE NOTES ADDED.

### CODE REVIEW

File: app/views/responsiblepeople/legal_name.scala.html [hunk 1]
```diff
@@ -32,13 +32,16 @@
 ) {
     @form(f, controllers.responsiblepeople.routes.LegalNameController.post(index, edit, flow)) {
 
+
+        <p>@Messages("responsiblepeople.legalnamechangedate.paragraph")</p>
```
Comment (drewmiley) [inline app/views/responsiblepeople/legal_name.scala.html]: This is correct?

Comment (slavvco) [reply app/views/responsiblepeople/legal_name.scala.html]: I think so, this page was marked to do both paragraph and hint, so I did it :)

Comment (drewmiley) [reply app/views/responsiblepeople/legal_name.scala.html]: Cool

**Comment by slavvco** [comment]
passed QA as per @mostafasina, merged with master and resolved conflicts

**Pull request merged**